assets/cart.js

```
@@ -0,0 +1,406 @@
+class TabList extends HTMLUListElement {
+  constructor() {
+    super();
+
+    this.controls.forEach((button) => button.addEventListener('click', this.handleButtonClick.bind(this)));
+  }
+
+  get controls() {
+    return this._controls = this._controls || Array.from(this.querySelectorAll('[aria-controls]'));
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)